### DISCUSSION TITLE
CSV Import with ; as an seperator

### DISCUSSION THREAD

**Discussion by techfull-de** [Budibase/budibase#1677] [Help]
Hi there 
Is there the possibility to import semicolon separated texts ?

texts often contain a comma...

i'm looking forward to answers maybe there is a solution.

best wishes 
Julian

**Comment by mjashanks** [comment]
Hey @techfull-de 

Your cells that contain a comma should be surrounded by quotes. e.g.

```
name,address
mike,"80 main street, new york"
```

If I open a spreadsheet with commas and save as CSV, then it is saved like this for me.

It will import correctly if surrounded by quotes.
